invenio_oauthclient/utils.py

```diff
@@ -160,27 +160,20 @@ def oauth_unlink_external_id(external_id):
                                      method=external_id['method']).delete()
 
 
-def is_local_url(target):
-    """Determine if URL is a local.
-
-    :param target: The URL to check.
-    :returns: ``True`` if the target is a local url.
-    """
-    server_name = current_app.config['SERVER_NAME']
-    test_url = urisplit(target)
-    return not test_url.host or test_url.scheme in ('http', 'https') and \
-        server_name == test_url.host
-
-
 def get_safe_redirect_target(arg='next'):
     """Get URL to redirect to and ensure that it is local.
 
     :param arg: URL argument.
     :returns: The redirect target or ``None``.
     """
     for target in request.args.get(arg), request.referrer:
-        if target and is_local_url(target):
-            return target
+        if target:
+            target = urisplit(target)
+            if target.path:
```

Comment:
Could this be an issue for redirects to the root path (ie. for `https://videos.cern.ch`, `target.path == ''`)?

Reply:
yes this should be integrated.